browser-test/src/trusted_intermediary.test.ts

```diff
@@ -330,9 +385,13 @@ test.describe('Trusted intermediaries', () => {
   test('dashboard contains required indicator note and optional marker', async () => {
```

Comment:
Nitpick: Should this test name be changed to not have "dashboard"?

Reply:
Ahh!! Dashboard is coming back to kick me!!!